libs/ledger-live-common/src/hw/getVersion.ts

```diff
@@ -77,6 +122,41 @@ export default async function getVersion(
       mcuVersionBuf = mcuVersionBuf.slice(0, mcuVersionBuf.length - 1);
     }
     mcuVersion = mcuVersionBuf.toString();
+
+    const isOSU = rawVersion.includes("-osu");
+
+    if (!isOSU) {
+      const deviceModel: DeviceModelId = deviceTargetIds[targetId.toString(16)];
+
+      if (isBootloaderVersionSupported(seVersion, deviceModel)) {
+        const bootloaderVersionLength = data[i++];
+        let bootloaderVersionBuf: Buffer = Buffer.from(
+          data.slice(i, i + bootloaderVersionLength)
+        );
+        i += bootloaderVersionLength;
+
+        if (bootloaderVersionBuf[bootloaderVersionBuf.length - 1] === 0) {
+          bootloaderVersionBuf = bootloaderVersionBuf.slice(
+            0,
+            bootloaderVersionBuf.length - 1
+          );
+        }
+        bootloaderVersion = bootloaderVersionBuf.toString();
```

Comment:
Could be interesting at one point to have a unit test on the new `bootloaderVersion`, `hardwareVersion` and `languageId` properties to check that the extraction is correct and do a kind of documentation (with a unit test, someone could see what is expected) ;) 

You can maybe create a fake/mocked `transport` to send whatever you want with `transport.send`

Something like
```
const mockedTransportSend = jest.fn();

// You need to mock all the function of Transport though, but it can be done quickly with jest.fn()
// We could create a "mock builder" for Transport too if it's used elsewhere
mockedTransport: Transport = {
  static readonly listen: jest.fn(),
  static readonly open: jest.fn(),
  close: jest.fn()
  on: jest.fn(),
  ...etc
  setExchangeUnresponsiveTimeout: jest.fn(),
  send: mockedTransportSend,
  ...etc
}

// And then in your test, resolvedValue because it's a Promise<typeof value>
mockedTransportSend.mockResolvedValue(value)
```